sdk/src/test/java/com/google/cloud/dataflow/sdk/io/BigQueryIOTest.java

```diff
@@ -338,7 +340,7 @@ public void testCustomSink() throws Exception {
         new TableRow().set("name", "b").set("number", 2),
         new TableRow().set("name", "c").set("number", 3)))
     .setCoder(TableRowJsonCoder.of())
-    .apply(BigQueryIO.Write.to("project-id:dataset-id.table-id")
+    .apply(BigQueryIO.Write.to("dataset-id.table-id")
```

Comment:
This is the tweak to the test you asked for -- implicitly, it tests that the project ID makes it to the sink because the sink will throw if the project is missing.